apps/src/tests/Test556.tsx

```diff
@@ -8,15 +8,20 @@ const Stack = createNativeStackNavigator();
 export default function App() {
   return (
     <NavigationContainer>
-      <Stack.Navigator>
+      <Stack.Navigator
+        screenOptions={{
+          animation: 'fade',
+          headerTitle: () => <Text>Simple Native Stack</Text>,
+          headerRight: () => <Text>Right</Text>,
+        }}>
         <Stack.Screen name="First" component={First} />
         <Stack.Screen name="Second" component={Second} />
       </Stack.Navigator>
     </NavigationContainer>
   );
 }
 
-function First({ navigation }) {
+function First({ navigation }: any) {
```

Comment:
Do you need to provide the return type at all? If so, `any` doesn't seem too good :D

Reply:
It's the type of the props, not the return type. + I don't think we need to define the types of the navigation prop in such minimal repro, that's why `any`.

Reply:
@alduzy But is it this `any` type required here? I believe that you can skip the `any` here and leave destructurized property without any type 😄

Reply:
https://github.com/software-mansion/react-native-screens/pull/2247/commits/779fd7c2571dfc6ffaf8aa054379c01dea991de4